web/src/elements/utils/eventEmitter.ts

```diff
@@ -9,16 +9,21 @@ export const isCustomEvent = (v: any): v is CustomEvent =>
 
 export function CustomEmitterElement<T extends Constructor<LitElement>>(superclass: T) {
     return class EmmiterElementHandler extends superclass {
-        dispatchCustomEvent(eventName: string, detail = {}, options = {}) {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
If the detail is a scalar or array, package it directly into the custom event, rather than including it with the target and such.  This is probably over-engineering, and I should probably just remove `target` at this point.  I think it's a YAGNI issue.